lib/auth/clt.go

```diff
@@ -130,12 +131,15 @@ func (c *Client) GetSessions() ([]session.Session, error) {
 }
 
 // GetSession returns a session by ID
-func (c *Client) GetSession(id session.ID) (*session.Session, error) {
+func (c *Client) GetSession(namespace string, id session.ID) (*session.Session, error) {
+	if namespace == "" {
+		return nil, trace.BadParameter("missing parameter namespace")
+	}
 	// saving extra round-trip
 	if err := id.Check(); err != nil {
 		return nil, trace.Wrap(err)
 	}
-	out, err := c.Get(c.Endpoint("sessions", string(id)), url.Values{})
+	out, err := c.Get(c.Endpoint("namespaces", namespace, "sessions", string(id)), url.Values{})
```

Comment:
We support both old and new UI without breaking any compatibility changes, see this block:

```go
	return httplib.RewritePaths(&srv.Router,
		httplib.Rewrite("/v1/nodes", "/v1/namespaces/default/nodes"),
		httplib.Rewrite("/v1/sessions", "/v1/namespaces/default/sessions"),
		httplib.Rewrite("/v1/sessions/([^/]+)/(.*)", "/v1/namespaces/default/sessions/$1/$2"),
	)
```